examples/lang/hetzner0.mcl

```diff
@@ -0,0 +1,12 @@
+# simple demo of the Hetzner resource
+hetzner "resourcetest" {
+	apitokenpath => "../mgmt-dev/apitoken.txt", # local file
+	servertype => "cx11",
+	datacenter => "hel1-dc2",
+	image => "debian-11",
+	state => "absent",
+	waitinterval => 5,
+	Meta:poll => 10,
```

Comment:
It's new, but stylistically sensible to leave a newline and put the Meta: stuff at the bottom.